### PR TITLE
Renumbering the PDB files using the Uniprot sequence

### PR DESCRIPTION
**Pull request by steven-albanese**
Opening up this PR to work from, since I messed up and commited to master originally

### CODE REVIEW

**Comment by steven-albanese** [comment]
Ok I've got this mostly working, but there are still two issues I need to address: 

1) Handle waters correctly. There is a heterogen tag that I can use to exclude these 
2) Fix how the caps are being handled, since they are seemingly randomly named 
3) Check if renumbering is necessary

**Comment by steven-albanese** [comment]
@jchodera the script for renumbering is working now! I just checked all of the renumbered structures and they look good (aside from the two axitinib structures, which I posted about in kinase).

File: hauser-abl-benchmark/receptors/cAbl-axitinib_nowat.pdb [hunk 1]
```diff
@@ -6690,12 +6690,12 @@ ATOM   4508  HZ2 LYS B 508     -77.146 -20.015 -47.913  1.00  0.00           H
 ATOM   4509  HZ3 LYS B 508     -76.943 -20.146 -46.285  1.00  0.00           H  
 ATOM   4510  HZ1 LYS B 508     -77.115 -21.511 -47.202  1.00  0.00           H  
 ATOM   4511  H   LYS B 508     -79.272 -17.426 -44.559  1.00  0.00           H  
-ATOM   4512  N   NMA B 508A    -79.526 -14.267 -47.348  1.00  0.00           N  
-ATOM   4513  CA  NMA B 508A    -79.586 -12.926 -47.918  1.00  0.00           C  
-ATOM   4514  H   NMA B 508A    -78.800 -14.866 -47.719  1.00  0.00           H  
-ATOM   4515 1HA  NMA B 508A    -80.498 -12.392 -47.649  1.00  0.00           H  
-ATOM   4516 2HA  NMA B 508A    -78.731 -12.338 -47.588  1.00  0.00           H  
-ATOM   4517 3HA  NMA B 508A    -79.540 -12.992 -49.006  1.00  0.00           H  
+ATOM   4512  N   NMA B 514    -79.526 -14.267 -47.348  1.00  0.00           N
```
Comment (jchodera) [inline hauser-abl-benchmark/receptors/cAbl-axitinib_nowat.pdb]: This doesn't look right---there's a missing space in the altloc field that is corrupting the PDB file format.

**Review by jchodera — DISMISSED** [review]
Looks like there might be something wrong in the removal of altlocs.

File: hauser-abl-benchmark/receptors/cAbl-axitinib_nowat.pdb (hunk 1 above)
Comment (steven-albanese) [reply hauser-abl-benchmark/receptors/cAbl-axitinib_nowat.pdb]: I think I know what's wrong

Comment (steven-albanese) [reply hauser-abl-benchmark/receptors/cAbl-axitinib_nowat.pdb]: Ok- so these aren't really altlocs. The C-terminal caps in Maestro are numbered as the last residue+A (i.e, 513A or 512A). My code was changing this to be last residue+1 (i.e., 513 or 514). I've reverted it back to be last residue+A. 

I had changed this to match up with the N-terminal caps, which are first residue -1. I'm fine with either numbering scheme

**Comment by steven-albanese** [comment]
New commit has the C-terminal caps back to the Maestro numbering scheme

**Comment by steven-albanese** [comment]
ponatinib structure has been fixed! Just waiting to hear back about the correct axitinib structure from Kevin

**Comment by steven-albanese** [comment]
@jchodera this should be good to go now!

**Pull request merged**